src/actions/checkForUpdate.js

```diff
@@ -6,9 +6,40 @@ import DependencyGraph from '../lib/DependencyGraph';
 
 import type {PackageJSONType} from '../utils';
 
-export default async function checkForUpdate(
-  packageDefinition: PackageJSONType
+export async function checkForUpdateViaArguments(
```

Comment:
Menurutku, ini bisa diabstraksi jadi lebih enak lagi sih. The `update` action should do a single thing, but with different interface.

misalnya, km bisa introduce konsep 'tiap actions harus punya `interactiveHandler` & `parameterHandler`':

```
createCheckForUpdate(...) {
   ...
   function checkForUpdate(..., ...) {...}

   function interactiveHandler(..., ...) {
      /** olah input dari inquirer buat jadi parameternya `checkForUpdate` **/
      const checker = checkForUpdate(..., ...)
      /** format outputnya pake inquirer **/
   }

   function parameterHandler(..., ...) {
      /** olah input dari args commander buat jadi parameternya `checkForUpdate` **/
      const checker = checkForUpdate(..., ...)
      /** format outputnya pake console.log/anything **/
   }

   return {
      interactiveHandler,
      parameterHandler,
   };
}
```

ntar di `index.js`, km bisa bikin layer buat semacam `routing` fungsi mana yg harus dipanggil ketika app dijalankan 😄